presto-main/src/main/java/com/facebook/presto/execution/scheduler/SqlQueryScheduler.java

```diff
@@ -951,18 +944,9 @@ else if (partitioningHandle.equals(SCALED_WRITER_DISTRIBUTION)) {
                         }
                     })
                     .collect(toImmutableSet());
-
-            this.childStageIds = children.stream()
```

Comment:
interesting. So previously this part of code is not useful?

Reply:
Yeah, it was a dead code